components/form/rangeDatepicker/src/index.js

```diff
@@ -63,6 +63,7 @@ class FormRangeDatepicker extends Component {
             onChange={this._handleChangeStart}
             className="sui-FormRangeDatepicker-input"
             locale={locale}
+            withPortal={this.props.showModal}
```

Comment:
what does mean "withPortal" ?
if this will display a modal, i would use "showModal"